pkg/connectors/protos/credentials.proto

```diff
@@ -0,0 +1,17 @@
+// Copyright 2020 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package connectors;
+option java_package = "com.datmesh";
+option go_package = "github.com/ibm/the-mesh-for-data/pkg/connectors/protobuf";
+
+message Credentials {
+    string access_key = 1; 	
+    string secret_key = 2;
+    string username = 3;
+    string password = 4;
+    string api_key = 5;
+	string resource_instance_id = 6;
```

Comment:
Indent issue

Reply:
Resolved indentation issue and documented a few fields in credentials.proto